src/cli/install.rs

```diff
@@ -15,18 +16,52 @@ pub struct Args {
     pub lock_file_usage: super::LockFileUsageArgs,
 
     #[arg(long, short)]
-    pub environment: Option<String>,
+    pub environments: Option<Vec<String>>,
 
     #[clap(flatten)]
     pub config: ConfigCli,
+
+    #[arg(long, short, conflicts_with = "environments")]
+    pub all: bool,
 }
 
 pub async fn execute(args: Args) -> miette::Result<()> {
     let project =
         Project::load_or_else_discover(args.manifest_path.as_deref())?.with_cli_config(args.config);
-    let environment = project.environment_from_name_or_env_var(args.environment)?;
 
-    get_up_to_date_prefix(&environment, args.lock_file_usage.into(), false).await?;
+    // Install either:
+    //
+    // 1. specific environments
+    // 2. all environments
+    // 3. default environment (if no environments are specified)
+    let envs = if let Some(envs) = args.environments {
+        envs
+    } else if args.all {
+        project
+            .environments()
+            .iter()
+            .map(|env| env.name().to_string())
+            .collect()
+    } else {
+        vec![project.default_environment().name().to_string()]
+    };
+
+    let mut installed_envs = Vec::with_capacity(envs.len());
+    for env in envs {
+        let environment = project.environment_from_name_or_env_var(Some(env))?;
+        installed_envs.push(environment.name().to_string());
+        get_up_to_date_prefix(&environment, args.lock_file_usage.into(), false).await?;
+    }
+
+    let s = if installed_envs.len() > 1 { "s" } else { "" };
+    // Message what's installed
+    eprintln!(
+        "> Installed environment{s}: {}!",
```

Comment:
```suggestion
        "> Installed environment{s}: {}",
```

Another idea would be to remove the next print completely and use this to share that the environment is ready

```suggestion
        "> The following environment{s} are ready to use: \n\t{}",
```

Reply:
Good idea, will change :)